react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -196,10 +242,12 @@ function _mapStateToProps(state, ownProps) {
             participantId);
     let avatar;
     let connectionStatus;
+    let participantName;
 
     if (participant) {
         avatar = getAvatarURL(participant);
         connectionStatus = participant.connectionStatus;
+        participantName = participant.name || 'Fellow Jitster';
```

Comment:
TODO: factor this out to a function, since it also needs to account for "(me)" and so on.

Reply:
There's a function called appendSuffix in the display-name feature that might do some of what you want.

Reply:
Thanks for the pointer!